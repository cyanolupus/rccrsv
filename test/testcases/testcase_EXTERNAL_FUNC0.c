int foo(); int testcase() { return foo(); }
