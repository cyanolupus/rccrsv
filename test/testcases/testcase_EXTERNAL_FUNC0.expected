Hello, World!
return 123
